dev/src/codewind/project/Project.ts

```diff
@@ -464,6 +465,57 @@ export default class Project implements vscode.QuickPickItem {
         });
     }
 
+    public async onLoadRunnerUpdate(event: {projectID: string, status: string, hcdPath: string, timestamp: string}): Promise<void> {
+        Log.d(`${this.name} load runner status changed to ${event.status}`, event);
+        const loadTestPath = `${this.localPath.fsPath}/load-test/`;
+        if (event.status == "hcdReady") {
+            Log.d("Health center data is ready to be copied to workspace");
+            if (!(await fs.existsSync(loadTestPath))) {
+                await fs.mkdirSync(loadTestPath);
+            }
+            await exec("docker inspect codewind-pfe", (error, stdout, stderr) => {
+                if (error) {
+                    Log.e(stderr);
```

Comment:
Shouldn't this error (and most of the other error blocks) be reported to the user?